micro-ui/web/micro-ui-internals/packages/modules/workbench/src/utils/index.js

```
@@ -191,17 +191,26 @@ const updateTitleToLocalisationCodeForObject = (definition, schemaCode) => {
   });
   return definition;
 };
-const formatDates = (value, type) => {
+const formatDates = (value, type,key) => {
   if (type != "EPOC" && (!value || Number.isNaN(value))) {
     value = new Date();
   }
   switch (type) {
     case "date":
-      return new Date(value)?.toISOString?.()?.split?.("T")?.[0];
+      const options = { timeZone: 'Asia/Kolkata', year: 'numeric', month: '2-digit', day: '2-digit' };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)